package.json

```diff
@@ -73,8 +73,8 @@
     "azure-arm-website": "0.11.2",
     "azure-arm-datalake-analytics": "0.4.3",
     "azure-arm-datalake-store": "0.4.2",
-    "azure-arm-batch": "0.2.1",
-    "azure-batch": "0.5.1",
+    "azure-arm-batch": "0.4.0",
```

Comment:
@amarzavery 
Just wanted to confirm: Should I be changing the top-level package.json now, or do you guys handle this after the packages have actually been published?